openid-sharedsignals-framework-1_0.md

```diff
@@ -1961,10 +1969,10 @@ that they will receive events related to the subject.
 
 ## Information Harvesting {#management-sec-information-harvesting}
 SETs may contain personally identifiable information (PII) or other non-public
-information about the event transmitter, the subject (of an event in the SET),
+information about the Event Transmitter, the subject (of an event in the SET),
```

Comment:
Just Transmitter?

Reply:
```suggestion
information about the Transmitter, the subject (of an event in the SET),
```

Reply:
i actually think i like the verbose-ness in this paragraph/